hera_cal/utils.py

```diff
@@ -263,7 +326,233 @@ def combine_calfits(files, fname, outdir=None, overwrite=False, broadcast_flags=
     uvc.write_calfits(output_fname, clobber=True)
 
 
+def get_miriad_times(filepaths, add_int_buffer=False):
+    """
+    Use aipy to get filetimes for Miriad file paths.
+    All times mark the center of an integration, which is not the standard Miriad format.
+
+    Parameters:
+    ------------
+    filepaths : type=list, list of filepaths
+
+    add_int_buffer : type=bool, if True, extend start and stop times by a single integration duration 
```

Comment:
I don't understand this parameter. What's the use case? What does the result represent in this case?

Reply:
its used for the binning. in the case where an LST bin falls between file boundaries, this is needed to let the `f_select` know to load in the boundary files. not completely necessary for the user to understand what this does, its all under the hood

Reply:
Fair enough. Perhaps just add a comment to the docstring explaining the primary use case?